[2021-06-27 05:01:24,205] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: csv_data_ingesation.process_csv 2021-06-27T04:59:53.982771+00:00 [queued]>
[2021-06-27 05:01:24,216] {taskinstance.py:655} INFO - Dependencies all met for <TaskInstance: csv_data_ingesation.process_csv 2021-06-27T04:59:53.982771+00:00 [queued]>
[2021-06-27 05:01:24,216] {taskinstance.py:866} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 05:01:24,216] {taskinstance.py:867} INFO - Starting attempt 1 of 2
[2021-06-27 05:01:24,216] {taskinstance.py:868} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 05:01:24,226] {taskinstance.py:887} INFO - Executing <Task(PythonOperator): process_csv> on 2021-06-27T04:59:53.982771+00:00
[2021-06-27 05:01:24,229] {standard_task_runner.py:53} INFO - Started process 2415 to run task
[2021-06-27 05:01:24,277] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: csv_data_ingesation.process_csv 2021-06-27T04:59:53.982771+00:00 [running]> ip-172-31-21-147.ec2.internal
[2021-06-27 05:01:24,334] {python_operator.py:114} INFO - Done. Returned value was: True
[2021-06-27 05:01:24,347] {taskinstance.py:1048} INFO - Marking task as SUCCESS.dag_id=csv_data_ingesation, task_id=process_csv, execution_date=20210627T045953, start_date=20210627T050124, end_date=20210627T050124
[2021-06-27 05:01:34,199] {logging_mixin.py:112} INFO - [2021-06-27 05:01:34,199] {local_task_job.py:103} INFO - Task exited with return code 0
